src/DIRAC/Interfaces/Utilities/DCommands.py

```diff
@@ -0,0 +1,658 @@
+#!/usr/bin/env python
+import os.path
+import re
+import uuid
+import stat
+import json
+import random
+from configparser import ConfigParser, NoSectionError, NoOptionError
```

Comment:
Do we need another config file (in a different format) lying around? There is already `~/.dirac.cfg`.

Reply:
Do any users actually use `~/.dirac.cfg`?